.github/workflows/integration_tests.yml

```diff
@@ -829,8 +759,9 @@ jobs:
           github_token: ${{secrets.GITHUB_TOKEN}}
           workflow: CI.yml
           run_id: ${{ needs.integration-test.outputs.ARTIFACT_RUN_ID }}
+          name: build-config
```

Comment:
Only download the `build-config` artifact instead of all artifact, since none of the other artifacts are actually needed here.